sql/codegen_xgboost_test.go

```diff
@@ -51,3 +58,17 @@ func TestXGBFiller(t *testing.T) {
 	a.NoError(err)
 	a.Equal(filler.ParamsCfgJSON, string(paramsJSON))
 }
+
+func TestXGBFillerPredict(t *testing.T) {
```

Comment:
Please also add a unit test in `executor_test` to test the pred SQL.

Reply:
Done.